{{cookiecutter.project_name}}/.github/workflows/test.yaml

```diff
@@ -40,28 +40,13 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
-      - name: Set up Python ${{ matrix.python }}
-        uses: actions/setup-python@v5
-        with:
-          python-version: ${{ matrix.python }}
-          cache: "pip"
-          cache-dependency-path: "**/pyproject.toml"
-
-      - name: Install test dependencies
-        run: |
-          python -m pip install --upgrade pip wheel
-      - name: Install dependencies
-        run: |
-          pip install ${{ matrix.pip-flags }} ".[dev,test]"
-      - name: Test
+      - name: Install Hatch
+        uses: pypa/hatch@install
+      - name: run tests using hatch
         env:
           MPLBACKEND: agg
           PLATFORM: ${{ matrix.os }}
           DISPLAY: :42
-        run: |
-          coverage run -m pytest -v --color=yes
-      - name: Report coverage
-        run: |
-          coverage report
+        run: hatch test --cover --python ${{ matrix.python }}
```

Comment:
Allegedly this creates a coverage report, so the `coverage report` command would not be needed.

Reply:
correct!